.github/scripts/prepare-release

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+command -v gh >/dev/null 2>&1 || {
+	printf 'need "gh" tool: https://github.com/cli/cli\n'>&2
+	exit 1
+}
+: ${BRANCH:=$(git branch --show-current)}
+while getopts hb: arg; do
+	case "$arg" in
+		b) BRANCH="$OPTARG";;
+		?) printf 'usage: %s: [-h] [-b BRANCH] version\n' "$0"
+			exit 2;;
+	esac
+done
+shift $(($OPTIND - 1))
```

Comment:
Any chance we could add remote as an arg for us self-namespace-dwellers?:
```sh
: ${REMOTE:=origin}
while getopts ":b:r": arg; do
	case "$arg" in
		b) BRANCH="$OPTARG";;
		r) REMOTE="$OPTARG";;
		?) printf 'usage: %s: [-h] [-b BRANCH] [-r REMOTE] version\n' "$0"
			exit 2;;
	esac
done
```
We can also merge and I can create a PR if you prefer